### PR TITLE
Create pylint.yml

### PR DESCRIPTION
**Pull request by ruwayd99**
Github action that adds a linting check with each commit.

### CODE REVIEW

**Comment by DSep** [comment]
Also check out #76

File: .github/workflows/pylint.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      uses: tj-actions/changed-files@v11.5
+    - name: Analysing the code with pylint
+      run: |
+        PYTHONFILES=""
+        for file in ${{ steps.changed-files.outputs.all_modified_files }}; do
+          if [[ -f $file && $file == *.py ]]
+            then
+              PYTHONFILES="${PYTHONFILES} $file"
+              echo $PYTHONFILES
+              pylint $PYTHONFILES
```
Comment (MarcoYuen17) [inline .github/workflows/pylint.yml:32]: Does this run pylint on the current file as well as previously seen files? I think it'd make more sense to just do `pylint $file`

Comment (MarcoYuen17) [reply .github/workflows/pylint.yml:32]: In this case you wouldn't need PYTHONFILES either, and you can change the echo to just echo the current file as well

Comment (ruwayd99) [reply .github/workflows/pylint.yml:32]: Thanks for letting me know! I'll change that then

**Review by MarcoYuen17 — APPROVED** [review]
Looks good to me. Let's merge this and try pushing problematic code to a new branch to see the output sometime

**Pull request merged**